modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -742,7 +532,6 @@ java.lang.IllegalArgumentException: Did not recognize index type VECTOR
 ----
 
 [TIP]
-====
 If a store copy is required, make a note of the information in the `createStatement` column returned from the `SHOW INDEX` command.
 For example:
 [source,cypher]
```

Comment:
same here as above, without the grouping the example isn't in the tip box which feels wrong as it is part of the tip